pages/blog/posts/post5.md

```diff
@@ -0,0 +1,163 @@
+  inventory management by automatically adjusting stock levels for both raw
+  materials and finished products.
+
+Link the receipt with the **Stock Entries** and the **Subcontracting Returns**
+ in **Doc References** to maintain clear documentation of the transactions
+ associated with the receipt. This ensures that all relevant movements of goods
+  are tracked accurately within the system.
+
+### Purchase Receipt
+
+It is used to formally receive the services performed on raw materials by the
+job worker, resulting in the production of finished items.
+
+## Subcontracting Scenarios
+
+### Resend the Finished Good for Rework
+
+When the finished goods are received back from the supplier and require some
+rework, they must be sent back to the supplier for further processing. This
+process can be effectively managed in ERPNext through Subcontract Returns, which
+allows for the documentation and tracking of items being returned to the
+subcontractor for additional modifications or adjustments.
+
+- From the **Subcontracting Receipt**, generate **Subcontract Returns** for the
+finished goods that need to be sent back to the supplier.
+
+![Subcontract Returns](../assets/subcontract_returns.png)
+
+- Create a stock entry return from the subcontracting order if any raw materials
+need to be sent back to the supplier along with the finished goods.
+- Link the stock entry against which the stock entry return is made in the **Doc
+References** section of the stock entry return. This provides a clear connection
+ between the original stock entry and the return, facilitating better tracking
+ and accountability of inventory movements.
+
+![Stock Entry Doc References](../assets/stock_entry_doc_references.png)
+
+- Finally, receive the returned goods through the subcontracting receipt to
+complete the process cycle.
+- Link the stock entries and subcontracting returns associated with the
+subcontracting receipt in the **Doc References**.
+
+![Stock Entry Doc References](../assets/subcontracting_receipt_doc_references.png)
+
+### Multilevel Subcontracting Process
+
+Multi-Level Subcontracting refers to a complex manufacturing process where
+multiple subcontractors are involved in the production of a final product. In
+this setup, primary subcontractors may outsource certain tasks to secondary
+ subcontractors, creating a layered supply chain.
+
+For Example, a furniture manufacturer, XYZ, produces unfinished wooden chairs
+ in-house and then outsources further processing. Subcontractor-A stains and
+  finishes the chairs, while Subcontractor-B assembles the cushions.
+
+Here's how the multi-level subcontracting process can be implemented in ERPNext:
+
+- **Initial Subcontracting**: Company XYZ initiates the process by creating a
+Purchase Order and a Subcontracting Order for **Subcontractor-A**. A stock entry
+is generated to transfer raw materials from XYZ to **Subcontractor-A**.
+
+- **Completion by Subcontractor-A**: Upon completion of the job work,
```

Comment:
Warehouse references. How this is handled in India Compliance App? how this is reported for ITC-04.

- Excess stock return.
- Sending it back for repairs.